docs/APi.puml

```diff
@@ -0,0 +1,200 @@
+@startuml ExpenseTracker
+
+package "API" {
+  class Expense {
+   - id : Guid
+   - amount : Decimal
+   - date : Date 
+   - description : String
+   - category : ExpenseCategory
+  }
+  
+  class ExpenseCategory {
+   - id : Guid
+   - Name : String
+  }
+  
+  class Income {
+   - id : Guid
+   - amount : Decimal
+   - date : Date 
+   - source : String
+  }
+  
+  class UserItems {
```

Comment:
I couldn't find who is using this class

Reply:
this class is used by ApplicationDbContext